test/datalog_tests/tutorial.ast.expected

```diff
@@ -235,6 +245,20 @@ HostIP(.host=host, .addr=addr) :- HostAddress(.host=host, .addrs=addrs), var add
 HostIPVSep(.host=host, .addrs=vaddrs) :- HostAddress(.host=host, .addrs=addrs), var vaddrs = vsep(split_ip_list(addrs)).
 Sum(.x=x, .y=y, .sum=(x + y)),
 Product(.x=x, .y=y, .prod=(x * y)) :- X(.x=x), X(.x=y).
+BestPrice(.item=item, .price=best_price) :- Price(.item=item, .vendor=_, .price=price), var best_price = Aggregate((item), std.group_min(price)).
+WorstPrice(.item=item, .price=best_price) :- Price(.item=item, .vendor=_, .price=price), var best_price = Aggregate((item), std.group_max(price)).
+BestVendor(.item=item, .vendor=best_vendor, .price=best_price) :- Price(.item=item, .vendor=vendor, .price=price), var vendor_price_vec = Aggregate((item), std.group2vec((vendor, price))), (var best_vendor, var best_price) = (var min_vendor = "";
+                                                                                                                                                                                                                                  (var min_price = (64'd18446744073709551615: bit<64>);
```

Comment:
looks like the indentation is off for these constructs.

Reply:
fixed